### ISSUE TITLE
Tooltip on toolbar options

### ISSUE THREAD

**Issue by whimsicaldreamer** [xotahal/react-native-material-ui#217]
Is it possible to add tooltips to options in the toolbar on long press?

**Comment by xotahal** [comment]
What do you mean? You can listen onLongPress a display whatever you want .. don't you?

**Comment by whimsicaldreamer** [comment]
Yes we can definitely do that but it would be fair enough to have it by default so that we can have the longPress free for other stuff. Right?

**Comment by xotahal** [comment]
I am not sure what are you talking about. Can you send me an URL to material design documentation with this tooltip on Toolbar?

**Comment by whimsicaldreamer** [comment]
I am not sure but you can find it on any android app. For ex: In whatsapp tap and hold any button on the toolbar and you can find a small tooltip being shown.

**Comment by xotahal** [comment]
I can imagine there is an app that uses this tooltips. But this library implements material design. I can't implement everything here :)

I know that MD has a tooltip component - https://material.io/guidelines/components/tooltips.html#tooltips-tooltips-mobile

And we can add it ... but it should be up to you where and when the tooltip will be shown ..

Correct?

**Comment by whimsicaldreamer** [comment]
That will be absolutely fine :)

**Comment by whimsicaldreamer** [comment]
And just a suggestion, it would be nice to expose a prop in the existing components which can be set to use the tooltip and also set the tooltip text.
